### DISCUSSION TITLE
Support for WooCommerce make an offer plugin

### DISCUSSION THREAD

**Discussion by kashif-malik12** [inspireui/support#5721] [New Feature/Ideas Requests]
this plugin is very important. We want to allow our customers to make an offer (specify quantity and price) and then go to offers page with push notification and start real time chat. The offer should go directly to the offers page and vendor chat and Allow finalize and accept offer or reject offer. Generate push notifications for all these. The accepted offer should directly go to cart of buyer and it should be valid for that quantity or more.

**Comment by osman-kocak** [comment]
@minhcasi
